### ISSUE TITLE
lib32-libstdc++5 moved to User Repository on Arch Linux

### ISSUE THREAD

**Issue by adtyanair** [flutter/flutter#25035]
```flutter doctor``` output prompts to install ```lib32-libstdc++5``` on Arch Linux using ```pacman -S lib32-libstdc++5``` but the package has been moved to AUR (https://aur.archlinux.org/packages/lib32-libstdc++5/). So the instruction shown by doctor is now wrong as the package can no longer be installed using pacman.

```
./bin/flutter doctor -v
[!] Flutter (Channel stable, v1.0.0, on Linux, locale en_IN.UTF-8)
    • Flutter version 1.0.0 at /opt/flutter
    • Framework revision 5391447fae (6 days ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)
    ✗ Downloaded executables cannot execute on host.
      See https://github.com/flutter/flutter/issues/6207 for more information
      On Debian/Ubuntu/Mint: sudo apt-get install lib32stdc++6
      On Fedora: dnf install libstdc++.i686
      On Arch: pacman -S lib32-libstdc++5


[✗] Android toolchain - develop for Android devices
    ✗ ANDROID_HOME = /opt/android-sdk
      but Android SDK not found at this location.

[✓] Android Studio (version 3.2)
    • Android Studio at /opt/android-studio
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[!] Connected device
    ! No devices available

! Doctor found issues in 3 categories.
```

**Comment by goderbauer** [comment]
Side note: We should be able to bundle our own libc++ with Flutter very soon. Once we do, there is no needs to install it anymore.

**Comment by kylesferrazza** [comment]
I was able to finally get it working with `pacman -S lib32-gcc-libs` - maybe this will work for you?

**Comment by adtyanair** [comment]
yeah, I was able to make it work with ```lib32-gcc-libs```

**Comment by kylesferrazza** [comment]
This is closed by e8d968ed8d76fb731aca238e7b45fe59ae623e55.
